layouts/partials/banner.html

```diff
@@ -1,11 +1,10 @@
 {{ if site.Params.show_banner -}}
 <div class="o-banner">
   <p>
-    Level up your API development with gRPC! Subscribe to the
-    <a href="https://www.youtube.com/@grpcio" target="_blank" rel="noopener">
-      <strong>gRPC YouTube channel</strong>
+    gRPConf 2024 is on Aug. 27th -
+    <a href="https://sessionize.com/grpconf2025" target="_blank" rel="noopener">
```

Comment:
Should this link work? :stuck_out_tongue:

Reply:
Thanks for catching that! Somehow 2024 turned into 2025 in the link.

Reply:
It could be a marketing trick, like with car models.